haystack/core/pipeline/async_pipeline.py

```diff
@@ -68,7 +69,8 @@ async def _run_component_async(  # pylint: disable=too-many-positional-arguments
                     raise PipelineRuntimeError.from_exception(component_name, instance.__class__, error) from error
             else:
                 loop = asyncio.get_running_loop()
```

Comment:
wdyt we leave a short comment here? Smth like:

Important: contextvars (e.g. active tracing Span) don’t propagate to running loop's ThreadPoolExecutor.
We use ctx.run(...) to preserve trace continuity.